gabbi/runner.py

```diff
@@ -120,8 +130,8 @@ def run_suite(handle, handler_objects, host, port, prefix, force_ssl=False,
 
     loader = unittest.defaultTestLoader
     test_suite = suitemaker.test_suite_from_dict(
-        loader, 'input', data, data_dir, host, port, None, None, prefix=prefix,
-        handlers=handler_objects)
+        loader, name, data, data_dir, host, port, None, None, prefix=prefix,
+        handlers=handler_objects, test_loader_name='gabbi-runner')
```

Comment:
without that the test name is `gabbi.suitemaker.some.stuff`. this makes it `gabbi-runner.some.stuff`.